packages/lit-html/src/lit-html.ts

```diff
@@ -609,9 +619,24 @@ export interface RenderOptions {
 
 /**
  * Renders a value, usually a lit-html TemplateResult, to the container.
- * @param value
- * @param container
- * @param options
+ *
+ * For example, to render "Hello, Zoe!" to the container `document.body`:
+ *
+ * ```js
+ * import {html, render} from 'lit';
+ *
+ * const name = "Zoe";
+ * render(html`<p>Hello, ${name}!</p>`, document.body);
+ * ```
+ *
+ * @param value Any renderable value, typically a {@linkcode TemplateResult}
```

Comment:
I think this is a good place to document renderable values, or at least include a link to the docs.

Reply:
Would the documentation link be: https://lit.dev/docs/templates/expressions/#child-expressions

Reply:
I think so.